changelog.d/14414.doc

```diff
@@ -0,0 +1 @@
+Edited sample Nginx reverse proxy configuration to use HTTP/1.1. Contributed by Brad Jones.
```

Comment:
Really minor stylistic nit: we usually write our changelogs in the imperative mood.
```suggestion
Edit sample Nginx reverse proxy configuration to use HTTP/1.1. Contributed by Brad Jones.
```